installer.py

```diff
@@ -1106,11 +1124,122 @@ def setup_osmosisd_service(osmosis_home):
     subprocess.run(["sudo", "mv", "osmosisd.service", unit_file_path])
     subprocess.run(["sudo", "systemctl", "daemon-reload"])
     subprocess.run(["systemctl", "restart", "systemd-journald"])
-    # Should we start the service?
-    # subprocess.run(["sudo", "systemctl", "start", "osmosisd"])
+    #Should we start the service?
+    subprocess.run(["sudo", "systemctl", "start", "osmosisd"])
     clear_screen()
 
+def branchSelection():  
+    version = LOCALOSMOSIS.version
+    print(bcolors.OKGREEN + """
+Would you like to run LocalOsmosis on the most recent release of Osmosis: {v} ?
+1) Yes, use {v} (recommended)
+2) No, I want to use a different version of Osmosis for LocalOsmosis from a branch on the osmosis repo
+3) No, I want to use a different version of Osmosis for LocalOsmosis from a branch on an external repo
+    """.format(
+        v=version) + bcolors.ENDC)
+
+    branchSelect = input(bcolors.OKGREEN + 'Enter Choice: ' + bcolors.ENDC)
+
+    if branchSelect == "1":
+        clear_screen() 
+    elif branchSelect == "2":
+        clear_screen()
+        #branchHandler() TODO
+    elif branchSelect == "3":
+        clear_screen()
+        #repoHandler() TODO
+    else:
+        clear_screen()
+        brachSelection()
+
+
+def installRust():
+    isRustInstalled = subprocess.run(
+        ["rustc --version"], capture_output=True, shell=True, text=True).stderr.strip()
+    if "not found" not in isRustInstalled:
+        return
+    print(bcolors.OKGREEN + """Rust not found on your device. Do you want to install Rust?:
+1) Yes, install Rust
+2) No, do not install Rust
+    """ + bcolors.ENDC)
+
+    installRust = input(bcolors.OKGREEN + 'Enter Choice: ' + bcolors.ENDC)
+
+    if installRust == "1":
+        clear_screen()
+        subprocess.run(
+            ["curl https://sh.rustup.rs -sSf | sh -s -- -y"], shell=True)  
+        clear_screen()
+    elif installRust == "2":
+        clear_screen()
+    else:
+        clear_screen()
+        installRust()
+
+# TODO: Cannot set this up. Cargo, rustup not found
```

Comment:
@niccoloraspa any suggestion on how i can resolve this issue? i made sure to run `installRust` before running `setupContactEnvironment` and yet rust is not recognized. I think it's not reading from the right path